autorest/preparer_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleCreatePreparer() {
 	} else {
 		fmt.Println(r.URL)
 	}
-	// Output: https://microsoft.com/a/b/c
```

Comment:
@garimakhulbe Why did you remove the "/microsoft.com"? The resulting URLs are not valid (they lack the required "//" after the HTTP scheme).